makefile

```diff
@@ -1,6 +1,10 @@
+ifeq (${BUILD},RELEASE)
+OPTIMIZATION=-O3
+endif
+
```

Comment:
We used to do only one build.  This made it difficult to use the debugger on the test binary.  The default now is to do a debug build, with OPTIMIZATION equal to -O3, the original value.  If the user builds with BUILD=DEBUG on the make line the -O3 is not used.  The resulting test binary, and any application used with the library built that way, are debuggable.